scripts/util/colors.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * Functions for manipulating RGB/RGBA strings and tuples.
+ */
+
+/** */
+type RgbaTuple = [number, number, number, number];
```

Comment:
Leave comment saying ranges

Reply:
[3] is ranged [0, 1], no?

Reply:
In tuple form, all values are `[0, 255]`. I think it's that way because that's how the colors are set (as cvars, and in color picker). Looking at the code, it could be reworked for `[0, 1]`, but the current way allows me to treat all the numbers the same for processing instead of dealing with both ints and a float.

Reply:
hey @tsa96 I just realized this comment reply didn't populate on the main conversation tab. Just pinging you incase it slipped through the cracks

Reply:
Woops, sorry!